package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jsc8",
-  "version": "0.17.11-update.1",
+  "version": "0.18.0",
```

Comment:
@dlozina-macrometa this will not correspond to the release cycle. 
Can we reset it to 0.1.1 or something or reset it to 0.17.11.1

Reply:
@dlozina-macrometa ☝️

Reply:
Yes, we will address it. When we merge all the changes, we will lift the version to 1.0.0 and decouple SDKs from the platform. Similar thing we did with UI. We will have the ability to have regular release cadence for fixing issues and updating features. @dwivedianurag  cc @elof